data/tooltips.txt

```diff
@@ -155,6 +155,9 @@ tip "jump speed:"
 tip "jump fuel:"
 	`How much fuel is consumed when you jump between systems using this drive.`
 
+tip "maintenance costs:"
+	`How many credits per day you need to spend to maintain this outfit. Even if a ship is parked or an outfit is in cargo, you still need to pay for its maintenance.`
```

Comment:
Not just outfits, maintenance costs can also apply to ship hulls.

Reply:
One will only ever be able to see this tooltip while looking at an outfit, though.

Reply:
The "Ship Info" panel also has some tooltips.
But you are right, the tooltip here is not visible in the "Ship Info" panel, so the text here is correct.